app/src/main/res/drawable/ic_baseline_history_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:autoMirrored="true" android:height="24dp"
```

Comment:
Any reason not to use the existing icon (`ic_restore_black_24dp`) that we were using for the History tab?